### ISSUE TITLE
schedule_viewer: &quot;No such file or directory: &quot;

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#19]
```
If you run schedule_viewer from a different directory you get an error such as 
IOError: [Errno 2] No such file or directory:
'schedule_viewer_files\\index.html'

Work around is to change into install directory and run it there.

I should fix it to make an absolute path relative to __file__
```

Original issue reported on code.google.com by `tom.brow...@gmail.com` on 1 May 2007 at 11:31

**Comment by GoogleCodeExporter** [comment]
```
Maybe event better than making an absolute path would be to embed the files in 
the exe.
```

Original comment by `tom.brow...@gmail.com` on 1 May 2007 at 11:59

**Comment by GoogleCodeExporter** [comment]
```
Hope this is enough to get everyone running:

schedule_viewer needs to find schedule_viewer_files. schedule_viewer_files is a
directory in our zip and source distributions.

By default schedule_viewer looks for schedule_viewer_files in the current 
working
directory. You can override this with the --file_dir argument.
```

Original comment by `tom.brow...@gmail.com` on 2 May 2007 at 12:09

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed by tom.brown.code-setup.py-fixes in transitfeed 1.0.8
```

Original comment by `tom.brow...@gmail.com` on 30 May 2007 at 7:40
- Changed state: **Fixed**
